libraries/chain/include/steem/chain/smt_objects/smt_token_object.hpp

```diff
@@ -90,7 +90,6 @@ class smt_token_object : public object< smt_token_object_type, smt_token_object
 
    /// set_setup_parameters
    bool              allow_voting = false;
```

Comment:
Should this default value be true or false? In #3158 I tried to make all of the default values match STEEM or at least be a common scenario. I think `allow_voting` makes more sense being initialized to true.